webapi/src/test/scala/org/knora/webapi/slice/admin/domain/service/ProjectExportServiceStub.scala

```diff
@@ -0,0 +1,30 @@
+package org.knora.webapi.slice.admin.domain.service
+
+import zio.Task
+import zio.ULayer
+import zio.ZIO
+import zio.ZLayer
+
+import java.nio.file.Path
+
+import org.knora.webapi.slice.admin.domain.model.KnoraProject
+
+object ProjectExportServiceStub {
+  val layer: ULayer[ProjectExportService] = ZLayer.succeed(new ProjectExportService {
+
+    /**
+     * Exports a project to a file.
+     * The file format is TriG.
+     * The data exported is:
+     * * the project metadata
+     * * the project's permission data
```

Comment:
I'm not quite sure but does it make sense to export the permission data without exporting the groups as well? Because sometimes the permissions refer to a group.

Reply:
Thanks for pointing it out. 

Currently "the project's permission data" actually means this query:
https://github.com/dasch-swiss/dsp-api/blob/main/webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/getProjectPermissions.scala.txt

I will have to get a better understanding of our current "getAllProjectData" implementation and what is necessary for the "export/import" functionality. AFAIU the goal currently is to be able to export the project, the users which are part of the project and all authentication/authorization information in such a way that when importing this data the state of the users and project is exactly the same as the time of the import.